xdsl/transforms/convert_onnx_to_linalg.py

```diff
@@ -67,6 +75,28 @@ def match_and_rewrite(self, relu: onnx.Relu, rewriter: PatternRewriter, /):
             linalg.YieldOp(max_op.result)
 
 
+@dataclass
+class ConstantOpLowering(RewritePattern):
+    @op_type_rewrite_pattern
+    def match_and_rewrite(self, constant: onnx.Constant, rewriter: PatternRewriter, /):
+        attr_value = list(constant.attributes.values())[1]
+        global_op = ml_program.Global(
```

Comment:
Need to insert this also, something like this:

```
SymbolTable.insert_or_update(module_op, func_op)
```

We don't have a best way to get the module op, probably just find it by getting parent_op until there are no more

Reply:
i don't understand why this is needed, what does it do?
q1. would it be something like: `SymbolTable.insert_or_update(constant, global_op)`
since constant is the "module op",  
q1b -  is this added to the apply bit instead, since i see that does moduleop
q2. "probably just find it by getting parent_op until there are no more" - recursive helper function right?
thanks

Reply:
q1: no, the first argument is the op in whose region the second argument will be inserted. Sorry, I changed the `op` in my copied code to module_op, but not the second arg, which should be `global_op`.
q1b: in the apply bit, we no longer have the data to insert, so it has to be done inside the pattern
q2: yeah recursive helper function is probably best. @math-fehr, do you have any ideas/advice for this or is it best to just manually roll one?

Reply:
@kayode-gif as a first step, just make it work, then we can make the code pretty :)